src/lib/components/modal/evm-verify-status/EvmVerifyAlert.tsx

```diff
@@ -0,0 +1,43 @@
+import { Alert, AlertDescription, Flex, Text } from "@chakra-ui/react";
+
+import { CustomIcon } from "lib/components/icon";
+import type { Option } from "lib/types";
+
+interface EvmVerifyAlertProps {
+  errorMsg: Option<string>;
```

Comment:
Do we really need to make it optional here? Can we && before calling this component?